fe1-web/store/reducers/SocialReducer.ts

```diff
@@ -124,12 +164,44 @@ export const makeChirpsList = () => createSelector(
       return [];
     }
     if (chirpList.byLaoId[laoId]) {
+      const store = chirpList.byLaoId[laoId];
       const allChirps: ChirpState[] = [];
-      chirpList.byLaoId[laoId].allIdsInOrder.forEach(
-        (id) => allChirps.push(chirpList.byLaoId[laoId].byId[id]),
+      store.allIdsInOrder.forEach(
+        (id) => allChirps.push(store.byId[id]),
       );
       return allChirps;
     }
     return [];
   },
 );
+
+const createReactionsEntry = (reactionByUser: Record<string, string[]>) => ({
+  '👍': reactionByUser['👍'] ? reactionByUser['👍'].length : 0,
+  '👎': reactionByUser['👎'] ? reactionByUser['👎'].length : 0,
+  '❤️': reactionByUser['❤️'] ? reactionByUser['❤️'].length : 0,
+});
+
+export const makeReactionsList = () => createSelector(
+  (state) => getSocialState(state),
+  (state) => getLaosState(state).currentId,
+  (list: SocialLaoReducerState, laoId: string | undefined):
+  Record<string, Record<string, number>> => {
+    if (!laoId) {
+      return {};
+    }
+    if (list.byLaoId[laoId]) {
+      const store = list.byLaoId[laoId];
+      const reactions: Record<string, Record<string, number>> = {};
+      store.allIdsInOrder.forEach(
+        (id) => {
+          const reactionByUser = store.reactionsByChirp[id];
+          if (reactionByUser) {
+            reactions[id] = createReactionsEntry(reactionByUser);
+          }
```

Comment:
I'm a bit surprised by the naming. Shouldn't this be ? 
```suggestion
          const chirpReactions = store.reactionsByChirp[id];
          if (chirpReactions) {
            reactions[id] = createReactionsEntry(chirpReactions);
          }
```